### PR TITLE
Add RWD to Banner, phone number and cart

### PR DESCRIPTION
**Pull request by Oskyyy**
Top bar:
-log in and register must be text-free (icons only)
-Dropdowns on the left are left unchanged
Banner:
-there should be a logo on the left
-the basket on the right (logo aligned up, i.e. slightly higher than it is)
-phon number and icon on the right(under the basket)

### CODE REVIEW

**Comment by Varenthein** [comment]
Something is wrong here:

image

**Comment by Oskyyy** [comment]
It should be fine now.

File: src/components/layout/CompanyClaim/CompanyClaim.module.scss [hunk 1]
```diff
@@ -69,4 +69,31 @@
...
   }
+
+  @media all and (max-width: 500px) {
+    img {
+      display: flex;
+      align-items: left;
+      margin: -80px 20px;
+    }
+    .cart {
+      .cartBox {
```
Comment (OskarLebuda) [inline src/components/layout/CompanyClaim/CompanyClaim.module.scss]: If we are using the SCSS - there is a simplest way to create a child element - try something like this:
```scss
.cart {
    &Box {
    }
}
```
this is the same, but more powerful :)

File: src/components/layout/CompanyClaim/CompanyClaim.module.scss [hunk 2]
```diff
@@ -69,4 +69,31 @@
...
+        display: inline-block;
+        position: relative;
+        color: #ffffff;
```
Comment (OskarLebuda) [inline src/components/layout/CompanyClaim/CompanyClaim.module.scss]: you can use a shortest version of color `#fff`

**Pull request merged**